packages/web/src/dev/ValidatorsList.tsx

```diff
@@ -264,19 +273,32 @@ class ValidatorsList extends React.PureComponent<Props, State> {
     }
 
     return (data || [])
-      .sort((a, b) => b.id - a.id)
       .sort((a, b) => compare(dAccessor(a), dAccessor(b)))
       .sort((a, b) => dir * compare(accessor(a), accessor(b)))
+      .sort((a, b) => this.isPinned(b) - this.isPinned(a))
+  }
+
+  isPinned({ address }: any) {
+    const list = (localStorage.getItem(localStoragePinnedKey) || '').split(',') || []
+    return +list.includes(address)
+  }
+
+  onPinned() {
```

Comment:
```suggestion
  onPinned = () =>  {
```

really all our class instance methods should be defined in this way.